docs/user-guide/platform-linux.md

```diff
@@ -78,6 +77,14 @@ cd pegasus-frontend-git
 makepkg -si
 ```
 
+!!! info "Steam Deck"
```

Comment:
I'm okay with making this into a `## Steam Deck` header, as _using_ Arch seems more of a technical background in this case, and I'm not sure the AUR part is relevant either.